### ISSUE TITLE
Add RHEL/CentOS 7 and 8 compatibility

### ISSUE THREAD

**Issue by ubellavance** [atqquebec/nwautomatedrecover#11]
When running on a RHEL 7 or 8 server, we get this error message:
```
message":"Incorrect JSON type for the field 'disks', please refer to the documentation. Details: Unexpected character ('\"' (code 34)): was expecting comma to separate Object entries
```

**Comment by ubellavance** [comment]
It probably has to do with the way we populate the $json variable:
```
json="{
\"recoverMode\": \"New\",
\"computeResourceMoref\": \"$clusterComputeResourceMoref\",
    \"datacenterMoref\": \"$datacenterMoref\" ,
\"datastoreMoref\": \"$recoverdatastore\",
\"vCenterHostname\": \"$vcenterserver\",
\"hostMoref\": \"$hostMoref\",
\"vmFolderMoref\": \"$vmFolderMoref\",
\"vmName\": \"$vmName\",
\"jobName\": \"$jobName\",
\"powerOn\": \"$powerOn\",
\"disks\": $disks,
    \"reconnectNic\": \"$reconnectNic\"
}"
```

**Comment by ubellavance** [comment]
In the REST API log:
```
Incorrect JSON type for the field 'disks', please refer to the documentation. Details: Unexpected character ('"' (code 34)): was expecting comma to separate Object entries
```

**Comment by bartoque** [comment]
I the code snippet it states the $disks variable as the only one that does not appear to have been put between escaped double quotes, like the other variables?

\"disks\": $disks,

typo

**Comment by ubellavance** [comment]
@bartoque it's not the problem but thanks for motivating me to put some efforts on this. I decided to use a json validator on the resulting json on RHEL7. Well, it showed me an error: 
```
Error: Parse error on line 14:
... 		"key": "2000" 		"name": "Hard disk 1
----------------------^
Expecting 'EOF', '}', ':', ',', ']', got 'STRING'
```
Found that I was missing a comma after the "2000".
After adding the missing comma, I got this error:
```
Error: Parse error on line 15:
... "Hard disk 1", 	}], 	"reconnectNic"
--------------------^
Expecting 'STRING', got '}'
```
Then I fixed the second error to find out that the order of the values in the disk array in the json is not the same and only the last element of the array doesn't need a comma at the end.

**Comment by ubellavance** [comment]
The problem is mostly in this part of the code:
```
curl -ks --location --request GET 'https://'$server':9090/nwrestapi/v3/global/vmware/vcenters/'$vcenterserver'/protectedvms/'$uuid'/backups/'$backupid'' \
 --header 'Authorization: Basic '$auth'' | jq -r '[.vmInformation.disks[]]' | egrep -v '(thinProvisioned|sizeInKb|datastoreName)' | sed -r 's/datastore-.*/'$recoverdatastore'"/g')
```
I guess that jq doesn't have the same behaviour in this version. RHEL 7's jq version is 1.5, while it is 1.3 on RHEL 6. It's also 1.5 on RHEL 8.

Raw output of the disk array on RHEL 6: 
```
[
  {
    "thinProvisioned": false,
    "sizeInKb": 104857600,
    "name": "Hard disk 1",
    "key": "2000",
    "datastoreName": "drlocalssds",
    "datastoreMoref": "datastore-132962"
  }
]
```
On RHEL7:
```
[
  {
    "datastoreMoref": "datastore-132962",
    "datastoreName": "drlocalssds",
    "key": "2000",
    "name": "Hard disk 1",
    "sizeInKb": 104857600,
    "thinProvisioned": false
  }
]
```

Well , it looks that jq makes alphabetical sort, but the order changed from version 1.3 to 1.5. It was a reverse order in 1.5, and it's now regular a-z in 1.5.

**Comment by ubellavance** [comment]
Fixed by commit acb8fa9
